app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -191,16 +191,17 @@ class _CustomChatInputState extends ConsumerState<CustomChatInput> {
                 child: Column(
                   mainAxisAlignment: MainAxisAlignment.start,
                   crossAxisAlignment: CrossAxisAlignment.start,
-                  children: [
-                    editMessage != null
-                        ? Consumer(builder: editMessageBuilder)
-                        : const SizedBox.shrink(),
-                    if (editMessage != null)
-                      _EditMessageContentWidget(
-                        convo: widget.convo,
-                        msg: editMessage,
-                      ),
-                  ],
+                  children: editMessage != null
+                      ? [
+                          Consumer(builder: editMessageBuilder),
+                          _EditMessageContentWidget(
+                            convo: widget.convo,
+                            msg: editMessage,
+                          ),
+                        ]
+                      : [
+                          const SizedBox.shrink(),
```

Comment:
`const SizedBox.shrink(),` doesn't requires here. Just `[]` should be sufficient.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1462/commits/738cb57c29a4ed3125aa82dadf2fb26b7eae993c